spec/requests/api/alert_statues_spec.rb

```diff
@@ -0,0 +1,99 @@
+describe "Alert Status API" do
+  let(:alert_status) { FactoryGirl.create(:miq_alert_status) }
+  let(:alert_status_url) { alert_statuses_url(alert_status.id) }
+  let(:container_provider) { FactoryGirl.create(:ems_container) }
+  let(:infra_provider) { FactoryGirl.create(:ems_infra) }
+  let(:vm) { FactoryGirl.create(:vm) }
+  let(:miq_alert_status_state) do
+    FactoryGirl.create(:miq_alert_status_state,
+                       :miq_alert_status => alert_status,
+                       :comment          => "Big problem",
+                       :user             => user,
+                       :assignee         => user)
+  end
+  let(:container_node) { FactoryGirl.create(:kubernetes_node, :name => "exmaple.com") }
+  let(:miq_alert) { FactoryGirl.create(:miq_alert, :severity => "info") }
+  let(:user) { FactoryGirl.create(:user_with_group) }
+
+  it "forbids access to actions without an appropriate role" do
+    api_basic_authorize
+
+    run_get(alert_statuses_url)
+
+    expect(response).to have_http_status(:forbidden)
+  end
+
+  it "reads all alerts statuses" do
+    api_basic_authorize collection_action_identifier(:alert_statuses, :read, :get)
+    alert_status
+    run_get(alert_statuses_url)
+    expect(response).to have_http_status(:ok)
+
+    alert_statues_amount = response.parsed_body["count"]
+
+    expect(alert_statues_amount).to eq(1)
```

Comment:
above 2 lines could just be the one, expect(response.parsed_body["count"]). to eq(1)